client/src/components/App.jsx

```diff
@@ -35,12 +77,50 @@ class App extends React.Component {
     }
   }
 
+  getItemOrder(itemIndex) {
+    const position = this.state.position;
+    const items = this.state.items.imgObjects.slice();
+    const numItems = items.length || 1;
+
+    if (itemIndex - position < 0) {
+      return numItems - Math.abs(itemIndex - position);
+    }
+    return itemIndex - position;
+  }
+
+  nextImage(event) {
+    event.preventDefault();
+    const position = this.state.position;
+    const items = this.state.items.imgObjects.slice();
+    const numItems = items.length || 1;
+    let newPosition = 0;
+    if (position === (numItems - 1)) {
+      newPosition = 0;
+    } else {
+      newPosition = position + 1;
+    }
+
+    this.setState({
+      position: newPosition
+    })
+  }
+
   render() {
     console.log(this.state.items)
```

Comment:
Don't forget to remove console logs when you're happily finished with a component